Corgibytes.Freshli.Cli/Functionality/Cache.cs

```diff
@@ -41,5 +47,40 @@ public static bool Prepare(DirectoryInfo cacheDir)
 
             return true;
         }
+
+        public static bool Destroy(DirectoryInfo cacheDir)
```

Comment:
This can be simplified significantly by just deleting the directory. We're going to have other commands adding lots of other files in here. I'm not sure it makes sense to keep track of every file that gets created.

Reply:
I've taken extensive trouble on the design because of some rather serious consequences of "just delete the directory". I'm going to be stubborn about this one, because it's a matter of preventing system-fatal mistakes.

https://github.com/corgibytes/freshli-cli/issues/50#issuecomment-1129255629

Reply:
Hmm... the home directory is an interesting scenario to defend against. in that case, if `freshli.db` is present, I say we delete everything. Otherwise, we refuse to do anything. The presence of `freshli.db` should be sufficient for determining that it's an app created directory.

Reply:
It shouldn't be sufficient, because the person could just as easily type...

```
freshli cache prepare --cache-dir /home/myself
```

...and then later...

```
freshli cache destroy --cache-dir /home/myself
```

I see two ways to proceed:

1. My approach.
2. Not allowing `cache prepare` in a non-empty directory unless it already contains `freshli.db`

Which would you prefer?

Reply:
In that case, let's go with option 2.